### ISSUE TITLE
Android in a Box

### ISSUE THREAD

**Issue by tpgxyz** [OpenMandrivaAssociation/distribution#2715]
Allow running Android apps (I have Anbox sort of working, but we can do better than the Brokenbuntu crowd...)

**Comment by benbullard79** [comment]
Checking if this is still active, I *think* it is.

**Comment by berolinux** [comment]
yes, still an interesting topic - the problem is that the upstream projects aiming at it tend to die quickly and it gets hard to chase the one that's working right now. RIP anbox, RIP shashlik, ...
